build.sbt

```diff
@@ -119,15 +121,16 @@ lazy val kinesisSettings =
   )
 
 lazy val kinesis = project
-  .settings(kinesisSettings)
+  .settings(kinesisSettings ++ dockerSettings)
   .enablePlugins(JavaAppPackaging, LauncherJarPlugin, DockerPlugin, BuildInfoPlugin)
   .dependsOn(core % "test->test;compile->compile")
 
 lazy val kinesisDistroless = project
   .in(file("distroless/kinesis"))
+  .settings(sourceDirectory := (kinesis / sourceDirectory).value)
   .settings(kinesisSettings ++ dockerSettingsDistroless)
   .enablePlugins(JavaAppPackaging, LauncherJarPlugin, DockerPlugin, BuildInfoPlugin)
-  .dependsOn(core % "test->test;compile->compile")
+  .dependsOn(kinesis % "test->test;compile->compile")
```

Comment:
Is this `dependsOn` line still needed?  It was needed before we changed the `sourceDirectory`, but I don't know if it's still needed now.

Reply:
We need both. Without `sourceDirectory`, the image gets built but it doesn't work as it doesn't have the collector code. Without `dependsOn`, the image can't get built because the module fails to compile.